apps/tmux/tmux.py

```diff
@@ -3,8 +3,17 @@
 ctx = Context()
 mod = Module()
 
+apps = mod.apps
+
+apps.tmux = "app.name: tmux"
```

Comment:
This line doesn't work for me using your tmux config on Linux (I do get the updated window title, but app.name doesn't match it). I'd actually be inclined to just leave it out completely since it only works if you've got Tmux set up a particular way.

I reckon the best way of enabling this is just for people to drop a file like this somewhere in their Talon user files:

`tmux_enabler.talon`
```
title: /^tmux/
-
tag(): user.tmux
```

Reply:
What would be the best way to communicate that? A comment or doc-block in this file, or an extension of the README? or something else?

Reply:
I reckon just put a short comment up near the top of the tmux.talon file to say the tag won't be enabled by default and suggesting how you might do it.